README.rst

```diff
@@ -50,6 +50,7 @@ Limitations
 
 - Multi-line FASTQ files are not supported.
 - FASTQ and BAM parsing is the focus of this library. The FASTA parser is not as optimized.
+- For reading and writing Zstandard (``.zst``) files, either the ``zstd`` command-line program or the Python ``zstandard`` package needs to be installed.
```

Comment:
Please add back the linebreak you had in an earlier comment. You just need to ensure you indent after the linebreak:

```
- First item
- Second
  item
- Third item
```

Reply:
Thanks, I'm not very familiar with RST